rhobs/recording/exporter_recording_rules.yml

```diff
@@ -0,0 +1,5 @@
+groups:
+- name: exporter
+  rules:
+  - record: konflux_up
+    expr: label_replace(grafana_ds_up, "service", "grafana_copy", "__name__", "(.+)")
```

Comment:
Why refer to the service as `grafana_copy` instead of `grafana`?

Reply:
@amisstea updated it.Please check.